OutOfSchool/OutOfSchool.DataAccess/Models/Provider.cs

```diff
@@ -0,0 +1,92 @@
+﻿#nullable enable
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using OutOfSchool.Services.Enums;
+
+namespace OutOfSchool.Services.Models
+{
+    public class Provider
+    {
+        public long Id { get; set; }
+
+        [Required(ErrorMessage = "Title is required")]
+        [DataType(DataType.Text)]
+        public string Title { get; set; }
```

Comment:
Shouldn't it be like this: public string Title { get; set; } = string.Empty ?

Reply:
We decided not to use this for properties with attribute Required